sci-libs/symengine/symengine-0.1.0.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE=""
+
+inherit cmake-utils python-single-r1
+
+DESCRIPTION="Fast symbolic manipulation library, written in C++"
+HOMEPAGE="https://github.com/sympy/symengine"
+SRC_URI="https://github.com/sympy/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="python threads openmp boost"
+
+DEPEND="
+	dev-libs/jemalloc
+	dev-libs/boost
+	python? ( dev-python/setuptools[${PYTHON_USEDEP}] )"
+RDEPEND="${DEPEND}
+	python? ( dev-python/numpy[${PYTHON_USEDEP}] )"
+
+CMAKE_BUILD_TYPE=Release
+
+src_configure() {
+	local mycmakeargs=(
+		-DCMAKE_INSTALL_PREFIX:PATH="${EPREFIX}"/usr \
+		$(cmake-utils_use_with boost) \
+		$(cmake-utils_use_with python)
+	)
+
+	if use openmp; then
+		if [[ $(tc-getCC) == *gcc ]] && ! tc-has-openmp; then
```

Comment:
Further if `openmp` is a requirement (i.e. it cannot be turned off) when `CC=gcc` it is really properly check in `pkg_setup` and there is no need for it in `IUSE`.